sql/size_utils.sql

```diff
@@ -42,8 +42,8 @@ BEGIN
                FROM (
                SELECT *, total_bytes-index_bytes-COALESCE(toast_bytes,0) AS table_bytes FROM (
                       SELECT
-                      sum(pg_total_relation_size('"' || c.schema_name || '"."' || c.table_name || '"'))::bigint as total_bytes,
-                      sum(pg_indexes_size('"' || c.schema_name || '"."' || c.table_name || '"'))::bigint AS index_bytes,
+                      sum(pg_total_relation_size(format('%%I.%%I', c.schema_name, c.table_name)))::bigint as total_bytes,
```

Comment:
Is the double `%` right? Haven't seen this before. What does it do?

Reply:
Yes, it is a % literal. This is a format call inside a format call so I want the outer format to print %I rather than try to replace it with an identifier.